database/src/main/resources/db/migration/V1.49.0__NOTIF-324_ensure-subtypes-are-lower-case.sql

```diff
@@ -0,0 +1,9 @@
+-- ensure the sub_types are lower case
+
+UPDATE endpoints
+    SET   endpoint_sub_type = LOWER(endpoint_sub_type)
+    WHERE endpoint_sub_type != LOWER(endpoint_sub_type);
+
+UPDATE notification_history
+    SET   endpoint_sub_type = LOWER(endpoint_sub_type)
+    WHERE endpoint_sub_type != LOWER(endpoint_sub_type)
```

Comment:
FYI, I just checked this with gabi:
| | Stage | Prod |
| - | - | - |
| SELECT COUNT(*) FROM endpoints WHERE endpoint_sub_type != LOWER(endpoint_sub_type) | 0 | 0 |
| SELECT COUNT(*) FROM notification_history WHERE endpoint_sub_type != LOWER(endpoint_sub_type) | 0 | 0 |

Reply:
Ah - then I can remove it:)

Reply:
@gwenneg enjoy while it lasts your Gabi days are counted ;-)

> expirationDate: '2022-05-01'

(we can renew)